### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by quentinleroy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,58 @@
 ![Logo Dark](https://github.com/camunda/issues/blob/main/resources/Logo_White_Github.png#gh-dark-mode-only)
 ![Logo Light](https://github.com/camunda/issues/blob/main/resources/Logo_Black.png#gh-light-mode-only)
 
-# Welcome to Camunda Issues 
+# Welcome to Camunda Public Roadmap 
 
-In this repository you can get an overview of key features we are currently working on (or planning to work on in the near future). Please don't hesitate to comment any questions or feedback you have directly on the respective issues.
+❇️ View the official [Camunda public product roadmap](https://github.com/orgs/camunda/projects/57).
+
+Our product roadmap is where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you. Have any questions or comments about items on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
```
Comment (MaxTru) [inline README.md:8]: How will we notice that an issue retrieved a new comment?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+
+The issues repository is for communicating Camunda’s roadmap. If you have feedback about the issues repository itself, such as how the issues are presented, let us know through general feedback using the appropriate GitHub issues template.
```
Comment (MaxTru) [inline README.md:10]: What do you want to express here? Shall the users create new issues for "meta-feedback"?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,58 @@
...
+
+## Guide to the roadmap
+Every item on the roadmap is an issue, with a label that indicates each of the following:
```
Comment (MaxTru) [inline README.md:13]: Let's deep-link to the issues.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,58 @@
...
+
+A feature area that indicates the component of the product to which the item belongs. For a list of current product component, see below.
+
+A target release version, indicating the version of the product in which we plan to deliver the feature. Camunda has minor releases twice a year, and alpha releases on a monthly basis. Once we know what version we are delivering a feature, we will update the issue to indicate that information.
```
Comment (MaxTru) [inline README.md:17]: Let's deep-link https://camunda.com/release-policy/

File: README.md (hunk 3 above)
Comment (MaxTru) [reply README.md:13]: Let's use a list for the following paragraphs.

Comment (MaxTru) [reply README.md:13]: Let's use some code-formatting for the `labels`

File: README.md [hunk 5]
```diff
@@ -1,6 +1,58 @@
...
+
+Once a feature is delivered, the version label will be applied to the roadmap issue and the issue status will be indicate if the feature was delivered in an alpha or minor version.
+
+Some of our features in "Researching" status may still be in the exploratory stages, and have no timeframe available. These are included in the roadmap only for early feedback. 
+
+### Roadmap phases
+The roadmap is arranged on a project board to give a sense for how far out each item is on the horizon. Every product or feature is added to a particular project board column according to the development phase in which it is. Be sure to read the disclaimer below since the roadmap is subject to change, especially further out on the timeline.
```
Comment (MaxTru) [inline README.md:24]: Let's deeplink the roadmap.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,58 @@
...
+- [Camunda BPMN JS](https://github.com/camunda/camunda-bpmn-js)
+- [Camunda DMN JS](https://github.com/camunda/camunda-dmn-js)
+- [Camunda Automation Platform 7](https://github.com/camunda/camunda-bpm-platform)
+
+### Participation
+
+Anyone can participate in the issues repository to share feedback with Camunda or show support for specific features. 
+
+- *Show support for specific feature*: to show your interest, you can write a comment in an existing issue or simply give a thumb up :+1:
+- *Propose a feature*: Use the issue template to propose a Feature Request. Before you do so, make sure to check the component repository (listed above) to find out if the feaure already exist
```
Comment (MaxTru) [inline README.md:53]: Let's add a deeplink to the issue template.

File: README.md [hunk 7]
```diff
@@ -1,6 +1,58 @@
...
+- *Report a bug*: Use the issue template to file a Bug Report. Before you do so, make sure to check the component repository (listed above) to find out if the bug has already been reported
```
Comment (MaxTru) [inline README.md:54]: Let's add a deeplink to the issue template.

Comment (MaxTru) [reply README.md:54]: Let's explain that the should use the respective componet repository to create the issue (if possible / available)

File: README.md [hunk 8]
```diff
@@ -1,6 +1,58 @@
...
+- *Change to the documentation*: Do you have suggestions to improve the documentation? Please write us using the Documentation Issue template provided and we will make sure to follow up with you.
```
Comment (MaxTru) [inline README.md:55]: Should they not better create an issue here https://github.com/camunda/camunda-platform-docs ?

File: README.md [hunk 9]
```diff
@@ -1,6 +1,58 @@
 ![Logo Dark](https://github.com/camunda/issues/blob/main/resources/Logo_White_Github.png#gh-dark-mode-only)
 ![Logo Light](https://github.com/camunda/issues/blob/main/resources/Logo_Black.png#gh-light-mode-only)
 
-# Welcome to Camunda Issues 
+# Welcome to Camunda Public Roadmap
```
Comment (MaxTru) [inline README.md:4]: Why rename to "Public Roadmap"?

Can we not keep it as "Issues" and explain that is serves 2 purposes:

* Highlight the roadmap
* Give the ability to open generic issues around Camunda Platform, that do not fit into any of the public component repositories?

**Review by MaxTru — CHANGES_REQUESTED** [review]
Please see my comments.

File: README.md (hunk 2 above)
Comment (quentinleroy) [reply README.md:10]: The intent here is to enable the user to provide feedback as much on the product as on the process to share the feedback itself. We want to continuously improve the way we interact with them and this feedback is important.

File: README.md (hunk 9 above)
Comment (quentinleroy) [reply README.md]: I don't have a strong opinion on this, we can keep the name of the repository as it is.

**Review by quentinleroy — COMMENTED** [review]
I reviewed all the changes requested and provided comments when necessary. The only changes left are related to the deep link towards the issue template, as they first need to be published.

**Review by MaxTru — CHANGES_REQUESTED** [review]
I still miss a section about what I am supposed to do with this repo.

Currently it states:
> The issues repository is for communicating Camunda’s roadmap. If you have feedback about the issues repository itself, such as how the issues are presented, let us know through general feedback using the appropriate GitHub issues template.

Can we please rework this and be clear about the key purposes of this repo being 2:

1. You can learn about the roadmap by looking at the epics on the board. If you have feedback, please create a comment.
2. If you have a bug to report or a suggestion to propose, then (a) visit the respective component repo and open an issue there or (b) if there is no such repo, open the issue here.

**Comment by quentinleroy** [comment]
The introduction states both main purposes of the repository:

> Our product roadmap is where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you. Have any questions or comments about items on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
> 
> The issues repository is for communicating Camunda’s roadmap. If you have feedback about the issues repository itself, such as how the issues are presented, let us know through general feedback using the appropriate GitHub issues template.

This repository's purpose is to communicate the roadmap and provide a way for the engineering teams to share private GIthub issues in a public repository, so they can be shared with externals. This repository **is not made** to collect feedback from external nor for them to create feature requests. However, since we cannot disable the possibility for users to open issues, we need instructions and a process to handle those. 

With this in mind, I do not want the introduction to explain in detail how they can create a feature request, but highlight the roadmap functionalities instead. The instructions to participate are placed at the end for this reason.

**Comment by quentinleroy** [comment]
Following our conversation this morning, I have made the changes in the introduction paragraph to highlight both use cases for this repository: communicate the roadmap and enable users to report bugs and improvement suggestions.

**Review by MaxTru — CHANGES_REQUESTED** [review]
I have some more suggestions. Thanks

File: README.md [hunk 10]
```diff
@@ -3,4 +3,58 @@
...
+
+2. Giving you a voice: If you have a bug to report or a suggestion to propose, then (a) visit the respective component repository listed below and open an issue there or (b) if there is no public repository, open the issue here.
+
+## Guide to the roadmap
+Every item on the roadmap is [an issue](https://github.com/camunda/issues/issues), with a `label` that indicates each of the following:
+
+- A `feature area` that indicates the component of the product to which the item belongs. For a list of current public product components, see below.
+
+- A `target release version`, indicating the version of the product in which we plan to deliver the feature. Camunda has minor releases twice a year, and alpha releases on a monthly basis. Once we know what version we are delivering a feature, we will update the issue to indicate that information. See our [release policy](https://camunda.com/release-policy/) for more information.
```
[the remark this answers is not in the document]
Comment (quentinleroy) [reply README.md:21]: Good point - I didn't know the exact difference myself. I have added a paragraph to detail it, based on available info in the Handbook: https://confluence.camunda.com/pages/viewpage.action?pageId=105513369#ProductDevelopmentProcess(PDP)-Potential

File: README.md [hunk 11]
```diff
@@ -3,4 +3,58 @@
...
+- [Camunda BPMN JS](https://github.com/camunda/camunda-bpmn-js)
+- [Camunda DMN JS](https://github.com/camunda/camunda-dmn-js)
+- [Camunda Automation Platform 7](https://github.com/camunda/camunda-bpm-platform)
+
+### Participation
+
+Anyone can participate in the issues repository to share feedback with Camunda or show support for specific features. 
+
+- *Show support for specific feature*: to show your interest, you can write a comment in an existing issue or simply give a thumb up :+1:
+- *Propose a feature*: Use the issue template to propose a Feature Request. Before you do so, make sure to check the component repository (listed above) to find out if the feature already exists
```
[the remark this answers is not in the document]
Comment (quentinleroy) [reply README.md]: Done

File: README.md [hunk 12]
```diff
@@ -3,4 +3,58 @@
...
+#### In Progress
+At this stage, we have understood the problem and we are actively developing a solution.
+
+#### Alpha: Primarily for testing and feedback
+Limited availability, requires pre-release agreement. Features still under development, and subject to change. Not recommended for production use, and limited documentation, SLAs or support provided.
+
+#### Minor: Generally available to all customers
+Ready for production use with associated SLA and technical support obligations.
+
+### Public Product Components
```
[the remark this answers is not in the document]
Comment (quentinleroy) [reply README.md:42]: Good catch - I've added it to the list.

**Review by quentinleroy — COMMENTED** [review]
Thanks @MaxTru for your proposition, I have implemented them. 
For the deep links missing, I was waiting for the template to be published and now all of them are in.
We should be all set now - please confirm and I will publish them. Thanks!

**Comment by quentinleroy** [comment]
> Some more suggestions. Maybe it makes sense to group the three labels (version, target, potential)

Thank you @MaxTru for your feedback. I have resolved all the comments and added the links where necessary. 

Let me know if you have anything else you'd like to change before we merge the changes. If not, I will merge it by end of the week.

File: README.md [hunk 13]
```diff
@@ -3,4 +3,61 @@
 
 # Welcome to Camunda Issues 
 
-In this repository you can get an overview of key features we are currently working on (or planning to work on in the near future). Please don't hesitate to comment any questions or feedback you have directly on the respective issues.
+❇️ View the official [Camunda public product roadmap](https://github.com/orgs/camunda/projects/57).
+
+The `camunda/issues` repository has two purposes:
+
+1. **Communicating our [product roadmap](https://github.com/orgs/camunda/projects/57)**: this is where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you. Have any questions or comments about items on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
```
Comment (MaxTru) [inline README.md:10]:
```suggestion
1. **Communicating our [product roadmap](https://github.com/orgs/camunda/projects/57)**: this is where you can learn about what features we're working on, what stage they're in, and when we expect to release them. Do you have any questions or comments about features on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
```

File: README.md [hunk 14]
```diff
@@ -3,4 +3,61 @@
...
+
+2. **Giving you a voice**: If you have a bug to report or a suggestion to propose, then (a) visit the respective [component repository](https://github.com/camunda/issues#public-product-components) listed below and open an issue there or (b) if there is no public repository, [open the issue here](https://github.com/camunda/issues/issues/new/choose).
+
+## Guide to the roadmap
```
Comment (MaxTru) [inline README.md:14]: Let's please make this consistent.
Currently you talk about `target:xyz` and also about `potential release version`

Comment (MaxTru) [reply README.md:14]: Additionally, you could use deep link like following to let users easily discover the concrete labels you are talking about:

Example for potential: https://github.com/camunda/issues/labels?q=potential

File: README.md [hunk 15]
```diff
@@ -3,4 +3,61 @@
...
+- [Camunda DMN JS](https://github.com/camunda/camunda-dmn-js)
+- [Camunda Automation Platform 7](https://github.com/camunda/camunda-bpm-platform)
+
+### Participation
+
+Anyone can participate in the issues repository to share feedback with Camunda or show support for specific features. 
+
+- *Show support for specific feature*: to show your interest, you can write a comment on an existing issue or simply give a thumb up :+1:
+- *Propose a feature*: Use the [issue template](https://github.com/camunda/issues/issues/new/choose) to propose a Feature Request. Before you do so, please check the [component repository](https://github.com/camunda/issues#public-product-components) listed above to find out if the feature already exists.
+- *Report a bug*: Use the issue template to file a Bug Report. Before you do so, make sure to check the component repository (listed above) to find out if the bug has already been reported; and open the bug in the appropriate repository whenever possible.
```
Comment (MaxTru) [inline README.md:59]: add deep link here as well for `listed above`

**Review by MaxTru — CHANGES_REQUESTED** [review]
Some more findings. Thanks

**Pull request merged**